app/components/trainees/filters/view.rb

```diff
@@ -32,8 +32,9 @@ def tags_for_active_filter(filter, value)
           [{ title: value, remove_link: remove_select_tag_link(filter) }]
         else
           value.each_with_object([]) do |v, arr|
+            titleHtml = tag.span("Remove ", class: "govuk-visually-hidden") + label_for(filter, v) + tag.span(" #{filter.humanize.downcase} filter", class: "govuk-visually-hidden")
```

Comment:
This probably deserves to be it's own method `def titleHtml(filter, v)` - you'll then be able to reuse it on line 32.